modules/consignment-api/outputs.tf

```diff
@@ -0,0 +1,11 @@
+output "database_url" {
+  value = aws_rds_cluster.consignment_api_database.endpoint
+}
+
+output "database_username" {
+  value = aws_rds_cluster.consignment_api_database.master_username
+}
+
+output "database_password" {
+  value = aws_rds_cluster.consignment_api_database.master_password
```

Comment:
I don't know if this applies to modules, but it looks like you can mark an output as sensitive so it's not printed in the console: https://www.terraform.io/docs/configuration/outputs.html#sensitive-suppressing-values-in-cli-output

Reply:
Not sure if it applies. I've added it in anyway, it can't hurt.